book/installation.rst

```diff
@@ -6,210 +6,206 @@
 
-You can clone the official `GitHub repository`_ to obtain the source code
-and checkout the release you want to use:
+**1a.1.** Go to the directory, to which you want to save the OroCRM folder ([$folder_location]) 
 
+**1a.2.** Use the *git clone* command. Specify the version to download. (In the example, it is "1.8.2").
+ 
 .. code-block:: bash
 
-    $ cd /var/www
+    $ cd [$folder_location]
     $ git clone -b 1.9.0 https://github.com/orocrm/crm-application.git orocrm
 
-.. note::
+.. hint:
 
     Along with ``1.9.0``, you can use any other released version or even the master branch to run
     the latest development version of the OroCRM.
 
-.. sidebar:: Installing the OroPlatform Application
+.. sidebar:: Get Source Code for the OroPlatform Application
 
-    Use the `Platform application repository URL`_ instead if you do not want to install the full
-    CRM:
+    If you are installing the Platform application, use the `Platform application repository URL`_ :
 
     .. code-block:: bash
 
-        $ cd /var/www
+        $ cd [$folder_location]
         $ git clone -b 1.9.0 https://github.com/orocrm/platform-application.git orocrm
 
-Next, you'll need to install the necessary dependencies which may take some time:
+**1a.3.** Go to the "orocrm" directory and run the *composer install* command, in order to install the dependencies. 
 
 .. code-block:: bash
 
+    $ cd orocrm
     $ composer install
+         
+.. _book-installation-github-clone-configuration-params:
+
+Configuration Parameters
+""""""""""""""""""""""""
 
-When Composer finished the installation of the dependencies, you will be asked to enter some
-system parameters needed to bootstrap the application:
+**1a.4.** At the end of the *composer install* command, you will be asked to enter some configuration parameters in 
+          the console. The parameters are required to bootstrap the application.  
+          
+          If you enter nothing, the default value (in brackets) will be used:
 
 .. code-block:: text
 
     Creating the "app/config/parameters.yml" file
     Some parameters are missing. Please provide them.
+    database_driver (pdo_mysql):
     database_host (127.0.0.1):
     database_port (null):
-    database_name (bap_standard):
+    database_name (oro_crm):
     database_user (root):
     database_password (null):
-    mailer_transport (mail):
+    mailer_transport (smtp):
     mailer_host (127.0.0.1):
     mailer_port (null):
     mailer_encryption (null):
     mailer_user (null):
     mailer_password (null):
-    websocket_host (127.0.0.1):
-    websocket_port (8080):
+    websocket_bind_address (0.0.0.0):
+    websocket_bind_port (8080):
+    websocket_frontend_host ('*'):
+    websocket_frontend_port (8080):
+    websocket_backend_host ('*'):
+    websocket_backend_port (8080):
     session_handler (session.handler.native_file):
     locale (en):
     secret (ThisTokenIsNotSoSecretChangeIt):
+    installed (null):
+    assets_version (null):
+  
```

Comment:
There is new parameter assets_version_strategy
    assets_version_strategy: time_hash # A strategy should be used to generate the global assets version, can be:
                                       # null        - the assets version stays unchanged
                                       # time_hash   - a hash of the current time
                                       # incremental - the next assets version is the previous version is incremented by one (e.g. 'ver1' -> 'ver2' or '1' -> '2')